### ISSUE TITLE
isp7  revisions

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#1094]
Tidying up some GO terms not to annotate lead me to

http://www.pombase.org/spombe/result/SPAC25B8.13c#recipext

 - [ ] Is annotated to TOR signalling, but it seems that it is a target of this pathway (affected by mutation in tor)

- [ ] I don't see any link between tor and this gene on the tor1 page? isp7 should be represented as a downstream target of TOR signalling in GO annotation

- [ ] I am trying to eradicate "cellular response to nitrogen starvation"
(only this one left), we should always be able to be more specific.
So we need to work out which part of this process is "in response to nitrogen starvation"
(this might be a bit difficult here...is TOR signalling in fission yeast always a response to nitrogen starvation?)

**Comment by mah11** [comment]
missing convenient detail: PMID:24344203 72f780a0a113152a

**Comment by mah11** [comment]
> [isp7] is annotated to TOR signalling

Ronit made that annotation, and I didn't (and don't) know of any reason to dispute their interpretation of the isp7 phenotypes. But I also don't have any specific reason to argue to keep it.

My best guess is that this reflects what they've put in the model in the last figure, where they show isp7 doing something to regulate TORC1 positively and TORC2 negatively, and that that part of the model is based on the Gad8 kinase activity stuff in figure 9. But it is a guess, and not a very informed one at that. I can't tell how much they know about what isp7 is actually doing to TORC1 or TORC2 at the molecular level, how direct the kinase regulation might be, etc. That's a big part of why I won't complain if you prefer to remove the annotation. 

> but it seems that it is a target of this pathway (affected by mutation in tor)

If isp7 is in the middle of a feedback loop it could affect and be affected by tor, but as I said, I'm basically trusting the authors.

> I don't see any link between tor and this gene on the tor1 page? isp7 should be represented as a downstream target of TOR signalling in GO annotation

isp7 is in an "affecting" extension on the decreased RNA level phenotype annotation. I didn't try to make any GO annotations for tor1 or tor2 because I wasn't sure the phenotypes were sufficiently specific. I don't see any kinase or other assays that would support any tor MF annotations (and it looks like we haven't configured any BP signaling terms to accept gene/gene product extensions).

But again, I also don't have reasons to oppose it if anyone better informed does think GO annotation is appropriate.


> I am trying to eradicate "cellular response to nitrogen starvation" ... is TOR signalling in fission yeast always a response to nitrogen starvation?

I have no idea. I know very little about TOR signaling in any species (and felt well out of my depth doing approval for this session!). Perhaps instead of an annotation to GO:0006995, we could add happens_during(GO:0006995) extensions to the import BP annotations Ronit also made?

**Comment by mah11** [comment]
tl;dr version: TOR stuff melts my head.

**Comment by ValWood** [comment]
Mine too, I'll take a look. Antonia probably knows more about this. I think of TOR as the  pathway downstream of TOR itself

"A series of molecular signals mediated by TOR (Target of rapamycin) proteins, members of the phosphoinositide (PI) 3-kinase related kinase (PIKK) family that act as serine/threonine kinases in response to nutrient availability or growth factors"

but if this is a feedback loop, it could be "positive regulation of TOR signalling...."

Won't worry about this part (I think GO need to address  consistency annotation of signalling pathways first).

I won't worry about the response to either until there is a consistent way to tie to the other processes
